bridges/otelzap/zap_test.go

```diff
@@ -0,0 +1,353 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Copyright (c) 2016-2017 Uber Technologies, Inc.
+package otelzap
```

Comment:
```suggestion
// Copyright (c) 2016-2017 Uber Technologies, Inc.

package otelzap
```

There needs to be a blank line. Otherwise it would render as package documentation.